chips/nrf5x/src/temperature.rs

```diff
@@ -15,7 +15,7 @@ use peripheral_interrupts;
 use peripheral_registers;
 
 /// Syscall Number
-pub const DRIVER_NUM: usize = 0x80_06_00_01;
+pub const DRIVER_NUM: usize = 0x60000;
```

Comment:
Aha! But actually we should just use the driver number constant in `capsules/src/temperature.rs` (which is correct) since that's the _actual_ system call driver being used. This module implements the HIL.

Reply:
Pending that, however, I'll confirm that this fix gets the temperature printing correctly for `sensors` :)